addons/pos_self_order/static/src/app/pages/landing_page/landing_page.js

```diff
@@ -57,6 +57,29 @@ export class LandingPage extends Component {
         return "";
     }
 
+    get draftOrder() {
+        return this.selfOrder.orders.filter((o) => o.access_token && o.state === "draft");
+    }
```

Comment:
You are only using this getter to get the length of the draft orders. Maybe return the length in the getter and not the orders